subscriptions-statistics-java/src/main/java/Main.java

```diff
@@ -0,0 +1,117 @@
+import java.util.concurrent.TimeoutException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class Main {
+    private static final Logger logger = LoggerFactory.getLogger(Main.class);
+        
+	public static void main(String[] args) throws Exception {
+            //region creating WorkspaceApi
+            //Creating a Statistics object with the apiKey, apiUrl
+            String apiKey = "<apiKey>";
+            String apiUrl = "<apiUrl>";
+            Statistics api = new Statistics(apiKey, apiUrl);
+            //endregion
+
+            //region Authentication
+            //Obtaining authentication token
+            String authUrl = String.format("%s/auth/v3", apiUrl);
+            ApiClient authClient = new ApiClient();
+            authClient.setBasePath(authUrl);
+            authClient.addDefaultHeader("x-api-key", apiKey);
+            authClient.getHttpClient().setFollowRedirects(false);
+            
+            logger.info("Retreiving token");
+            AuthenticationApi authApi = new AuthenticationApi(authClient); 
+			
+            String agentUsername = "<agentUsername>";
+            String agentPassword = "<agentPassword>";
+            String clientId = "<clientId>";
+            String clientSecret = "<clientSecret>";
+
+            String authorization = "Basic " + new String(Base64.getEncoder().encode(String.format("%s:%s", clientId, clientSecret).getBytes()));
+            DefaultOAuth2AccessToken resp = authApi.retrieveToken("password", authorization, "application/json", "*", clientId, null, agentUsername, agentPassword);
+            
+            String token = resp.getAccessToken();
+            logger.info("Initializing (token: {})", token);
+            //endregion
+            
+            //region Registering notifications handlers
+            //Here we register handlers for Service State (onServiceChange) and Statistics Update (onValues) notifications.
+            api.addListener(new Statistics.StatisticsListener() {
+                @Override
+                public void onServiceChange(ServiceState state) {
+                    logger.info("State: {}", state);
+                }
+
+                @Override
+                public void onValues(Collection<StatisticValue> list) {
+                    logger.info("Stats: {}", list);
+                }
+            });
+            
+            //region Initializing Statistics
+            //Initializing Statistics with a obtained earlier authentication token
+            CompletableFuture<Void> future = api.initialize(token);
+            
+            try {
+                future.get(30, TimeUnit.SECONDS);
+                //endregion
+                
+                
+                //region Creating subscriptions to existing agent statistics
+                StatisticDesc[] statistics = new StatisticDesc[] { 
+                    new StatisticDesc(UUID.randomUUID().toString(), agentUsername, "Agent", "CurrentAgentState"),
```

Comment:
The objectId for agent is employeeId, not agentUserName. Usually it's equal to username, but it can be configured (in config server) to have different value. I think the corresponding info should be added to comments.